amdsharktuner/tests/constraint_generator_test.py

```diff
@@ -391,7 +391,9 @@ def test_adjust_problem_size_for_pipeline(
         k=[3],
         batch=[0],
     )
-    taf_pipeline = iree_codegen.DispatchLoweringPassPipeline.LLVMGPUTileAndFuse
+    tile_and_fuse_pipeline = (
+        iree_codegen.DispatchLoweringPassPipeline.LLVMGPUTileAndFuse
+    )
```

Comment:
Can we use this constant directly instead of creating a local variable for it? I think it only hurts readability here